test/integration/targets/yum/tasks/yum.yml

```diff
@@ -135,6 +145,15 @@
     that:
         - "not yum_result is changed"
 
+- name: install sos with state latest idempotence with config file param
+  yum: name=sos state=latest
+  register: yum_result
```

Comment:
is
` config_file: /etc/yum.conf`
missing from this test?

Reply:
@tamsky it shouldn't be since that's the default one in the container instance as it executes in shippable, I'm just passing it to exercise that arg passing but I realized that because of a change in Ansible 2.7 I need to conditionalize it.

Reply:
@tamsky I'm a doof, it absolutely is. Sorry, I misunderstood.